CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# How to Contribute to Codecademy Community Bot
+
+All input is welcomed! Below are some of the basic steps to contribute to this project. Please read all of it to make sure you stay consistent with everybody else.
+
+Please review the _Issues_ tab and self-assign or ask to be assigned to one if you want to try to help in those. This keeps the efforts tidy and we know who is working on what.
+
+## Initial Setup
+
+If you don't have it already, install/setup Git on your machine.
+
+### Clone This Repo
+
+1. In the main page of the repo, notice the big green button that says "Code". Click it.
+2. Using the HTTPS or SSH option, copy the URL.
+3. If you are using HTTPS, using Git Bash/Terminal/Bash, navigate to the folder where you want this repo to be located in your machine and use:
+
+```
+$ git clone pasteURLYouJustCopied
+```
+
+4. If you are using SSH, follow this [GitHub tutorial](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+
+### Node.js and npm
+
+1. Install Node.js; it can be downloaded from [here](https://nodejs.org/en/).
```

Comment:
Since this project specifically requires Node 16.x I think we should mention that upfront here.